ui/vendor/_directional.scss

```diff
@@ -0,0 +1,63 @@
+// directional-scss | Author: Tyson Matanich (http://matanich.com), 2013 | License: MIT
+$dir: ltr !default;
+
+// Default $dir if not valid
+@if $dir != ltr and $dir != rtl {
+    $dir: ltr;
+}
+
+@function if-ltr($if, $else: null) {
+  @if $dir != rtl {
+    @return $if;
+  }
+  @else {
+    @return $else;
+  }
+}
+
+@function if-rtl($if, $else: null) {
+  @return if-ltr($else, $if);
+}
+
+$left: if-ltr(left, right);
+$right: if-ltr(right, left);
```

Comment:
Thanks for the feedback - In all honesty I'm not an rtl pro. I based this patch of the following article: http://matanich.com/2013/09/06/rtl-css-with-sass from which the above file was taken.

I'm trying to get my head around how what you're proposing in general would work for the components when it comes to things like margins, paddings, etc. For text I can see that a `slds-text-align--start` would work, but what about flipping the layout of say the docked-composer: https://github.com/salesforce-ux/design-system/pull/438/files/8d3423da2129117c8cc5cd383966b18efa231763#diff-0258d2cd8c34aac1a2fac1c7a226e2e7

Would it be better to provide an override rule such that if the parent element has `[dir=rtl]` then we flip padding/margins? That would keep things in one css file too.